website/styles/pages/blueprint.css

```
@@ -0,0 +1,443 @@
+    background: var(--bg-surface);
+    border-right: 1px solid var(--glass-border);
+    overflow-y: auto;
+    flex-shrink: 0;
+}
+
+.blueprint-toolbar {
+    padding: 20px;
+}
+
+.toolbar-section {
+    margaregin-bottom: 30px;
```

Biome (2.3.13)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)